PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/StorageUploadWithoutLengthCheck.java

```diff
@@ -0,0 +1,125 @@
+package com.microsoft.azure.toolkit.intellij.azure.sdk.buildtool;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.JavaRecursiveElementWalkingVisitor;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.PsiExpression;
+import com.intellij.psi.PsiExpressionList;
+import com.intellij.psi.PsiMethodCallExpression;
+import com.intellij.psi.PsiNewExpression;
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.jetbrains.annotations.Nls;
+import org.jetbrains.annotations.NotNull;
+
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+
+
+/**
+ * This class extends the LocalInspectionTool and is used to inspect the usage of Azure Storage upload APIs in the code.
+ * It checks if the upload methods are being called without a 'length' parameter of type 'long'.
+ * The methods to check are defined in a JSON configuration file.
+ * If such a method call is detected, a problem is registered with the ProblemsHolder.
+ */
+
+public class StorageUploadWithoutLengthCheck extends LocalInspectionTool {
+    @Nls
+    @NotNull
+    @Override
+    public String getDisplayName() {
+        return "Ensure Storage APIs use Length Parameter";
+    }
+
+    private static final List<String> METHODS_TO_CHECK = getMethodsToCheck();
+
+    private static List<String> getMethodsToCheck() {
+        try {
+            String configFileName = "META-INF/ruleConfigs.json";
```

Comment:
Make this a static final class constant.

Reply:
done:)